.github/workflows/regression-report.yml

```diff
@@ -34,7 +48,12 @@ jobs:
     if: |
       github.event.issue.pull_request
            && (startsWith(github.event.comment.body, 'Github, generate report for ')
-              || startsWith(github.event.comment.body, 'GitHub, generate report for '))
+              || startsWith(github.event.comment.body, 'GitHub, generate report for ')
+                 || startsWith(github.event.comment.body, 
```

Comment:
All `||` should aligned vertically on same indentation

Reply:
done, let me know if formatting is right